src/config/index.ts

```diff
@@ -148,6 +148,7 @@ export const TRUST_INTERRUPT_URL = '/' + TRUST_INTERRUPT_PAGE;
 
 // UPDATE templates
 export const UPDATE_DUE_DILIGENCE_OVERSEAS_ENTITY_PAGE = "due-diligence-overseas-entity";
+export const UPDATE_SIGN_OUT_PAGE = "update-sign-out";
```

Comment:
I thought we updated this to "sign-out" the view name seems have been updated to just sign-out.html .... does this even build/work?